chainer/functions/connection/convolution_3d.py

```diff
@@ -0,0 +1,279 @@
+import numpy 
+from six import moves
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import conv
+from chainer.utils import type_check
+
+
+if cuda.cudnn_enabled:
+    cudnn = cuda.cudnn
+    libcudnn = cuda.cudnn.cudnn
+    _cudnn_version = libcudnn.getVersion()
+    _fwd_pref = libcudnn.CUDNN_CONVOLUTION_FWD_SPECIFY_WORKSPACE_LIMIT
+    if _cudnn_version >= 4000:
+        _bwd_filter_pref = \
+            libcudnn.CUDNN_CONVOLUTION_BWD_FILTER_SPECIFY_WORKSPACE_LIMIT
+        _bwd_data_pref = \
+            libcudnn.CUDNN_CONVOLUTION_BWD_DATA_SPECIFY_WORKSPACE_LIMIT
+
+
+def _triplet(x):
+    if hasattr(x, '__getitem__'):
+        return x
+    return (x, x, x)
+
+
+def im32col_cpu(img, kh, kw, kd, sy, sx, sz, ph, pw, pd, pval=0, cover_all=False):
```

Comment:
`im2col_[cg]pu` and `col2im_[cg]pu` are implemented in `utils/conv`. So I think it is appropriate to implement 3d-version there, too.